examples/can.rs

```diff
@@ -1,61 +1,49 @@
-#![feature(used)]
+#![no_main]
 #![no_std]
 
-#[macro_use]
-extern crate cortex_m;
+extern crate cortex_m_rt;
+extern crate embedded_types;
+extern crate panic_halt;
 extern crate s32k144;
 extern crate s32k144evb;
-extern crate embedded_types;
 
-use cortex_m::asm;
+use cortex_m_rt::entry;
 
-use s32k144evb::{
-    can,
-    wdog,
-    spc,
-};
+use s32k144evb::{can, spc, wdog};
```

Comment:
Is this the correct format according to rustfmt?

Same on lines below.

Reply:
Yes, this was formatted via `cargo fmt`.